src/main/java/com/microsoft/azure/spring/data/documentdb/repository/support/SimpleDocumentDbRepository.java

```diff
@@ -48,10 +48,8 @@ public SimpleDocumentDbRepository(DocumentDbEntityInformation<T, ID> metadata,
         Assert.notNull(entity, "entity must not be null");
 
         // create collection if not exists
-        documentDbOperations.createCollectionIfNotExists(entityInformation.getCollectionName(),
-                entityInformation.getPartitionKeyFieldName(),
-                entityInformation.getRequestUnit(),
-                entityInformation.getIndexingPolicy());
+        documentDbOperations.createCollectionIfNotExists(this.entityInformation,
+                this.entityInformation.getPartitionKeyFieldName());
```

Comment:
Same as above, seems we can already get partitionKeyFieldName from entityInformation.